Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -503,41 +503,80 @@ namespace AzToolsFramework
 
         void AssetBrowserTreeView::DeleteEntries()
         {
-            auto entries = GetSelectedAssets(false); // do not include products, you cannot delete those!
+            auto entries = GetSelectedAssets(false); // you cannot rename product files.
 
-            if (entries.empty())
-            {
-                return;
-            }
```

Comment:
This was added to prevent a crash caused by no assets being selected when you press delete. Has something else fixed that crash since?

Reply:
Oh, nevermind me! I see that entries is being looped over instead of being used right away as before.

Reply:
😜